ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,281 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.5.0-beta1 27-Jan-2025
+Changes from 4.4.4
+1) New autopilots supported
+    - ACNS-F405AIO
+    - Airvolute DCS2 onboard FMU
+    - Aocoda-RC-H743Dual
+    - BrainFPV RADIX 2 HD
+    - CAN-Zero
+    - CM4Pilot
+    - CubeRed
+    - Esp32-tomte76, esp32nick, esp32s3devkit
+    - FlyingMoonH743
+    - Flywoo F405 Pro
+    - FlywooF405S-AIO with alternative IMUs
+    - Here4 GPS as flight controller
+    - Holybro 6X revision 6
+    - Holybro6X-45686 with 3x ICM45686 IMUs
+    - JAE JFB110
+    - KakuteH7 using ICM42688
+    - PixFlamingo (uses STM32L4PLUS CPU)
+    - PixPilot-C3
+    - PixSurveyA1-IND
+    - QiotekAdeptF407
+    - Sierra TrueNavIC
+    - SPRacing H7RF
+    - SW-Nav-F405
+    - YJUAV_A6
+    - YJUAV_A6SE, YJUAV_A6SE_H743
+2) Autopilot specific changes
+    - 1MB boards lose features to save flash (Payload Place, some battery monitors, NMEA Output, bootloaders, Turtle mode)
+    - CubeOrangePlus supports IMU high resolution sampling (works with ICM42688, ICM42652, ICM42670, ICM45686 IMUs)
+    - F4 processors with only 1 IMU gain Triple Harmonic Notch support
+    - F765-SE bdshot support on 1st 4 pin
+    - F7 and H7 boards lose DMA on I2C ports (not required, limited DMA better used elsewhere)
+    - FlyingMoonH743, FlyingMoonF427 2nd and 3rd IMU order swapped
+    - HEEWING-F405 supports CRSF
+    - MatekL431-RC bootloader added, DMA used for RC and GPS
+    - PH4-mini, PH4-mini-bdshot, Swan-K1 and TBS-Colibri-F7 BRD_SER4_RTSCTS param conflict fixed
+    - Pixhawk6C supports BMI088 baro
+    - TMotorH743, Heewing-F405 serial parameter definitions fixed
+3) AHRS/EKF enhancements and fixes
+    - AHRS_OPTIONS supports disabling fallback to DCM
+    - BARO_ALT_OFFSET slews more slowly (was 20sec, now 1min)
+    - EKF2 removed (can be re-enabled with Custom build server)
+    - External AHRS support for multiple GPSs
+    - InertialLabs INS-U external AHRS support
+    - Lord external AHRS renamed to MicroStrain5
+    - MAV_CMD_EXTERNAL_POSITION_ESTIMATE supports setting approximate position during dead-reckoning
+    - Microstrain7 external AHRS support
+4) Driver enhancements
+    - 3DR Solo supports up to 6S batteries
+    - Airspeed health checks vs GPS use 3D velocity
+    - BDShot support on IOMCU (e.g. main outputs of boards with IOMCU)
```

Comment:
I think you should say "dshot on all IOMCU channels on all boards with an IOMCU (specifically all Cubes) and bdshot on boards with F103-based IOMCUs (Pixhawk 6X)"
You can also say that dshot commands (e.g. motor reversal abd beeps) and EDT supported on IOMCU

Reply:
OK, I've changed the single line to these three lines.  I'm very open to changes in wordings of course, it helps if those are very specific.

- BDshot on boards with F103-based IOMCUs (e.g. Pixhawk 6X)
- Dshot on all IOMCU channels on all boards with an IOMCU (e.g. all CubePilot autopilots)
- Dshot commands (e.g. motor reversal abd beeps) and EDT supported on IOMCU

Reply:
Looks good!, One minor change:

BDshot on the first four channels of boards with F103-based IOMCUs (e.g. Pixhawk 6X)

Reply:
cool, fixed!